src/main/scala/viper/gobra/reporting/DefaultErrorBackTranslator.scala

```diff
@@ -61,12 +61,13 @@ object DefaultErrorBackTranslator {
       //      case vprrea.ReceiverNull(offendingNode) =>
       //      case vprrea.DivisionByZero(offendingNode) =>
       case vprrea.NegativePermission(CertainSource(info)) =>
-        NegativePermissionError(info)
+        NegativePermissionReason(info)
+      case vprrea.ReceiverNotInjective(CertainSource(info)) =>
+        ReceiverNotInjectiveReason(info)
```

Comment:
Sorry for being picky on this one, but I once added all reasons (as comments) from Silver and in the same order as they appear in Silver's codebase. Thus, I suggest to move this case to old line 69 (where the comment originally was)

Reply:
Done